sdk/storage/storage-file/package.json

```diff
@@ -63,17 +64,18 @@
     "typescript": "^3.2.2"
   },
   "scripts": {
-    "unit-node": "npm run build && npm run test:node",
-    "unit-browser": "npm run build && npm run test:browser",
-    "test": "npm run test:node && npm run test:browser",
-    "test:node": "cross-env TS_NODE_COMPILER_OPTIONS=\"{\\\"module\\\": \\\"commonjs\\\"}\" nyc mocha --compilers ts-node/register --require source-map-support/register --reporter mocha-multi-reporters --reporter-options configFile=mocha.reporter.config.json --full-trace --no-timeouts test/*.test.ts test/node/*.test.ts",
-    "test:browser": "karma start --single-run",
     "build": "npm run build:es6 && npm run build:nodebrowser && npm run build:browserzip && npm run build:test",
-    "build:test": "rollup -c rollup.test.config.js",
-    "build:nodebrowser": "rollup -c",
     "build:es6": "tsc -p tsconfig.json",
-    "build:autorest": "autorest ./swagger/README.md --typescript --use=@microsoft.azure/autorest.typescript@2.2.1",
+    "build:nodebrowser": "rollup -c",
     "build:browserzip": "gulp zip",
+    "build:test": "rollup -c rollup.test.config.js",
+    "build:autorest": "autorest ./swagger/README.md --typescript --use=@microsoft.azure/autorest.typescript@2.1.3",
```

Comment:
Why autorest version updated from 2.2.1 to 2.1.3?

Reply:
Good catch. It escaped my attention.
I wanted to ask regarding this.

Current master - 

Storage-blob uses autorest 2.1.3
https://github.com/Azure/azure-sdk-for-js/blob/2ce3366a2a2a064c584b9c459ad061672e0d6e01/sdk/storage/storage-blob/package.json#L76

Storage-file uses autorest 2.2.1
https://github.com/Azure/azure-sdk-for-js/blob/2ce3366a2a2a064c584b9c459ad061672e0d6e01/sdk/storage/storage-file/package.json#L75

Can you let me know if there are any issues in using the same autorest version(2.2.1) in both the packages?

Reply:
Update - changed this back to 2.2.1

Reply:
Different versions of autorest may potentially generate different code. We usually update this version number after a code re-generation.